demo/docbot/cluster_bootstrap.py

```diff
@@ -293,6 +293,56 @@ def initialize_index(client: MLClient) -> None:
     else:
         raise Exception("Failed to create index.")
 
+def ml_cleanup(client: MLClient) -> None:
+  """
+    Args:
+        client: MLCLient
+    Returns:
+        returns None
+    """
+  # Undeploy model
+  undeploy_model_response = client.undeploy_model(MODEL_STATE['model_id'])
+  if undeploy_model_response:
+      print('\nUndeploying model:')
+      print(undeploy_model_response)
+  else:
+      raise Exception("Failed to undeploy model.")
+  
+  # Delete model
+  delete_model_response = client.delete_model(MODEL_STATE["model_id"])
+  if delete_model_response:
+      print('\nDeleting model:')
+      print(delete_model_response)
+  else:
+      raise Exception("Failed to delete model.")
+
+  # Delete model connector
+  delete_connector_response = client.delete_connector(MODEL_STATE['connector_id'])
+  if delete_connector_response:
+      print('\nDeleting connector:')
+      print(delete_connector_response)
+  else:
+      raise Exception("Failed to delete connector.")
+
+  # Delete ingest pipeline
+  delete_pipeline_response = client._client.ingest.delete_pipeline(id='cohere-ingest-pipeline')
+  if delete_pipeline_response:
+      print('\nDeleting pipeline:')
+      print(delete_pipeline_response)
+  else:
+      raise Exception("Failed to delete ingest pipeline.")
+
+  # Delete index
+  delete_index_response = client._client.indices.delete(index='cohere-index')
+  if delete_index_response:
+      print('\nDeleting index:')
+      print(delete_index_response)
+  else:
+      raise Exception("Failed to delete index.")
+
+  # Clear MODEL_STATE
+  for state in MODEL_STATE:
```

Comment:
Here for clearing model state, lets instead take each of the parameters and assign their values to None. that way if there are multiple tests that depend on this data structure they can simply re-populate it.